session/pingpong/exchange_messaging.go

```diff
@@ -64,8 +65,9 @@ func (es *ExchangeSender) Send(em crypto.ExchangeMessage) error {
 		AgreementTotal: em.AgreementTotal,
 		Provider:       em.Provider,
 		Signature:      em.Signature,
-	})
-	_, err := es.ch.Send(p2p.TopicPaymentMessage, pMessage)
+	}
+	log.Info().Msgf("Sending P2P message to %q: %s", p2p.TopicPaymentMessage, pMessage.String())
```

Comment:
I suppose all these struct dumps should not be in info level

Reply:
Changed to debug level.